theia-rust-docker/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN cp -R /root/theia-cpp-extension/packages $THEIA_RUST_APP_PATH/.
 RUN cp -Rv /root/theia-cpp-extension/dev-packages/* $THEIA_RUST_APP_PATH/packages/.
 RUN . /root/nvm_setup.sh && cd $THEIA_RUST_APP_PATH && yarn --cache-folder $THEIA_RUST_APP_PATH/ycache 
 RUN . /root/nvm_setup.sh && cd $THEIA_RUST_APP_PATH && yarn theia build
+RUN . /root/nvm_setup.sh && cd $THEIA_RUST_APP_PATH && yarn theia download:plugins 
```

Comment:
We could merge all RUN commands into a single RUN to make the image smaller.

Reply:
done